_python/utils/utilities.py

```diff
@@ -31,9 +30,9 @@ def process_folder(root_folder, PageDirectories, process_md_file):
 # url: www.gointerject.github.io/wAbout/Basics-of-Report-Formulas.html
 def convert_filepath_to_url(file_path):
     """Convert a local file path to a www.gointerject.com URL with .html extension."""
-    relative_path = os.path.relpath(file_path, ROOT_FOLDER)  # Get relative path from root_folder
-    web_url = os.path.join("https://docs.gointerject.com", relative_path)  # Prepend the base URL
-    web_url = web_url.replace("\\", "/")  # Convert backslashes to forward slashes for URLs
+    relative_path = Path(file_path).relative_to(ROOT_FOLDER)  # Get relative path from root_folder
+    web_url = Path("https://docs.gointerject.com") / relative_path  # Prepend the base URL
```

Comment:
i dont think pathlib works for url paths, only file paths.

you might want to look at using `urllib.parse`

```python
from urllib.parse import urljoin

base_url = "https://example.com/path/to/"
relative_path = "resource/page.html"

full_url = urljoin(base_url, relative_path)
print(full_url)
```